api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -23,7 +22,13 @@ message RedisProxy {
   // Name of cluster from cluster manager. See the :ref:`configuration section
   // <arch_overview_redis_configuration>` of the architecture overview for recommendations on
   // configuring the backing cluster.
-  string cluster = 2 [(validate.rules).string.min_bytes = 1];
+  //
+  // .. attention::
+  //
+  //   This field is deprecated. Use a :ref:`catch-all
+  //   route<envoy_api_field_config.filter.network.redis_proxy.v2.RedisProxy.RouteConfig.catch_all>`
+  //   instead.
+  string cluster = 2 [(validate.rules).string.min_bytes = 1, deprecated = true];
```

Comment:
I think this can no longer be required, same for settings. We will need an app level check to make sure one or the other is set (until we remove these fields). Also, can you add these two fields to DEPRECATED.md?